setup.py

```diff
@@ -52,6 +53,14 @@ def read_file(filename, encoding='utf8'):
     'zope.interface',
 ]
 
+# Add pywin32 on Windows platforms to handle low-level system calls.
+# This dependency needs to be added using environment markers to avoid its installation on Linux.
+# However environment markers are supported only with setuptools >= 36.2.
+# So this dependency is not added for old Linux distributions with old setuptools,
+# in order to allow these systems to build certbot from sources.
+if StrictVersion(setuptools_version) >= StrictVersion('36.2'):
+    install_requires.append('pywin32 ; sys.platform == \'win32\'')
```

Comment:
I think you should change this to `sys_platform` or `platform_system` (at which point `win32` needs to be `Windows`). See the table at https://www.python.org/dev/peps/pep-0508/#environment-markers or the example in the `setuptools` docs at https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-platform-specific-dependencies.

Reply:
Done.